### ISSUE TITLE
set_dtbs_error_model

### ISSUE THREAD

**Issue by SimonBuatois** [pharmpy/pharmpy#405]
Would be good to add details in the documentation around the DTBS distribution: e.g. lambda and zeta parameters.

**Comment by sardnar** [comment]
+1 Here. I doesn't seem that actually this function transform DV in the dataset, only the error model. More clarification is needed, please.

**Comment by rikardn** [comment]
Agree. The DV transformation is done by the model and not in the dataset.
